.github/workflows/dapr-perf.yml

```diff
@@ -349,6 +349,9 @@ jobs:
       GOARCH: amd64
       AZURE_STORAGE_ACCOUNT: ${{ secrets.PERF_AZURE_STORAGE_ACCOUNT }}
       AZURE_STORAGE_ACCESS_KEY: ${{ secrets.PERF_AZURE_STORAGE_KEY }}
+      DAPR_PERF_METRICS_PROMETHEUS_PUSHGATEWAY_URL: ${{ secrets.DAPR_PERF_METRICS_PROMETHEUS_PUSHGATEWAY_URL }}
```

Comment:
Please confirm that they are added into GITHUB ENV as well.

Reply:
@ASHIQUEMD Have these secrets been added and are they part of Github ENV?

Reply:
These will be added, once we create the resources in Azure OSS subscription, we will be doing it this week.

Reply:
Can the secret be added first and then we can merge this PR?

Reply:
Yeah sure, waiting for [this PR](https://github.com/dapr/test-infra/pull/212) to be merged.